.circleci/config.yml

```diff
@@ -350,6 +350,17 @@ jobs:
       - store_artifacts:
           path: /home/circleci/.npm/_logs/
           destination: npm-logs
+      - run:
+          name: 'Create Incident on PagerDuty'
+          when: on_fail
+          command: |
+            curl -X POST --header 'Content-Type: application/json' -d '{
```

Comment:
Could / Have we set up a trial run of this CURL request? Would like to confirm before we proceed with merging.